lib/net_suite/client.rb

```diff
@@ -44,6 +52,17 @@ def #{method}(url = nil, body = nil, headers = nil, &block)
 
     attr_accessor :retry_count
 
+    def trace_call(method, url, request_payload, &)
+      return yield unless datadog_request_tracing?
+
+      ::Datadog::Tracing.trace("netsuite #{method}",
```

Comment:
Do we need the leading colons?
```suggestion
      Datadog::Tracing.trace("netsuite #{method}",
```

Reply:
Hmm, I thought I had gotten test errors locally without them, but I don't see them any longer. Fixed in https://github.com/careofvitamins/net_suite/pull/4/commits/687c812d554af06e8de3c9e227e03ea5f9a5de53